.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
```

Comment:
I think we'd also need this to run on PRs against master

Reply:
As it, a pull request that targets master? I think that, as it is, it'll run all PRs (e.g. it's running on this PR)